guitar-api.yaml

```diff
@@ -0,0 +1,145 @@
+openapi: "3.0.0"
+info:
+    version: "1.0.0"
+    title: "Guitar API"
+    description: Reference basic information on important guitar models
+
+servers:
+    - url: "http://localhost:8080"
+paths:
+    /guitar:
+        get:
+            summary: Return all guitars
+            description: Allows you to retrieve all guitars
+            operationId: getGuitars
+            responses:
+                200:
+                    description: successfully returned all guitars
+                    content:
+                        application/json:
+                            schema:
+                                type: array
+                                items: 
+                                    $ref: "#/components/schemas/guitar"
+                            examples:
+                                guitars:
+                                    value: {
+                                        $ref: "#/components/schemas/guitar"
```

Comment:
Need to review and address this: a) whether it is necessary and if so, b) why this doesn't pull in the guitar schema in the Swagger documentation.

Reply:
Solved this :)